example/android/app/build.gradle

```diff
@@ -28,8 +28,8 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "de.jonasbark.stripepaymentexample"
-        minSdkVersion 19
+        applicationId "jonasbark.de.stripepaymentexample"
+        minSdkVersion 21
```

Comment:
this would be a breaking change I'm afraid

Reply:
I revert it back. It is also runnable with 19

Reply:
I revert it back. It is also runnable with 19